frameworks/g-cloud-9/questions/services/lot.yml

```diff
@@ -5,12 +5,21 @@ options:
   - label: Cloud hosting
     value: cloud-hosting
     description: >
-      Used to be IaaS and PaaS.
+      Cloud hosting services are cloud platform or infrastructure services that can help buyers:
+
```

Comment:
Don't think you need an empty line before bullets, but will work OK like this too.